editoast/src/core/v2/pathfinding.rs

```diff
@@ -35,6 +40,36 @@ pub struct PathfindingRequest {
     pub rolling_stock_supported_signaling_systems: Vec<String>,
 }
 
+#[derive(Debug, Clone, Serialize, Deserialize, ToSchema)]
+pub struct RangeOffset {
+    start: u64,
+    end: u64,
+}
+
+#[derive(Debug, Clone, Serialize, Deserialize, ToSchema)]
+pub struct IncompatibleElectrification {
+    range: RangeOffset,
+    value: String,
+}
+
+#[derive(Debug, Clone, Serialize, Deserialize, ToSchema)]
+pub struct IncompatibleLoadingGauge {
```

Comment:
I'd add "Range" to the name (same for the 3)
```suggestion
pub struct IncompatibleLoadingGaugeRange {
```